definitions/src/lib.rs

```diff
@@ -90,40 +96,42 @@ pub enum FreeStmtOption {
     /// statement again with the same or different parameter values. If no cursor is open, this
     /// option has no effect for the application. `SQLCloseCursor` can also be called to close a
     /// cursor.
-    Close = 0,
+    SQL_CLOSE = 0,
     // SQL_DROP = 1, is deprecated in favour of SQLFreeHandle
     /// Sets the `SQL_DESC_COUNT` field of the ARD to 0, releasing all column buffers bound by
     /// `SQLBindCol` for the given StatementHandle. This does not unbind the bookmark column; to do
     /// that, the `SQL_DESC_DATA_PTR` field of the ARD for the bookmark column is set to NULL.
     /// Notice that if this operation is performed on an explicitly allocated descriptor that is
     /// shared by more than one statement, the operation will affect the bindings of all statements
     /// that share the descriptor.
-    Unbind = 2,
+    SQL_UNBIND = 2,
     /// Sets the `SQL_DESC_COUNT` field of the APD to 0, releasing all parameter buffers set by
     /// `SQLBindParameter` for the given StatementHandle. If this operation is performed on an
     /// explicitly allocated descriptor that is shared by more than one statement, this operation
     /// will affect the bindings of all the statements that share the descriptor.
-    ResetParams = 3,
+    SQL_RESET_PARAMS = 3,
 }
 
 /// Represented in C headers as SQLSMALLINT
+#[allow(non_camel_case_types)]
 #[repr(i16)]
 #[derive(Debug, PartialEq, Eq, Clone, Copy)]
 pub enum HandleType {
-    Env = 1,
-    Dbc = 2,
-    Stmt = 3,
-    Desc = 4,
+    SQL_HANDLE_ENV = 1,
+    SQL_HANDLE_DBC = 2,
+    SQL_HANDLE_STMT = 3,
+    SQL_HANDLE_DESC = 4,
 }
 
 /// Options for `SQLDriverConnect`
+#[allow(non_camel_case_types)]
 #[repr(u16)]
 #[derive(Debug, PartialEq, Eq, Clone, Copy)]
 pub enum DriverConnectOption {
-    NoPrompt = 0,
-    Complete = 1,
-    Prompt = 2,
-    CompleteRequired = 3,
+    SQL_DRIVER_NO_PROMPT = 0,
+    SQL_DRIVER_COMPLETE = 1,
+    SQL_DRIVER_PROMPT = 2,
+    SQL_DRIVER_COMPLETE_REQUIRED = 3,
 }
 
 // Attribute for string lengths
```

Comment:
Could you add the SQL prefix and remove the comment for IS_POINTER to IS_SMALLINT?
The values are in sqlext.h